### PR TITLE
Install yq the old fashioned way

### PR DESCRIPTION
**Pull request by ericwill**
Signed-off-by: Eric Williams <ericwill@redhat.com>

### What does this PR do?
Install yq using wget

### What issues does this PR fix or reference?

Hopefully unblocks the release this time.

### PR Checklist

[As the author of this Pull Request I made sure that:](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#pull-request-template-and-its-checklist)

- [x] [The Eclipse Contributor Agreement is valid](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-eclipse-contributor-agreement-is-valid)
- [x] [Code produced is complete](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-produced-is-complete)
- [x] [Code builds without errors](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-builds-without-errors)
- [ ] [Tests are covering the bugfix](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#tests-are-covering-the-bugfix)
- [x] [The repository devfile is up to date and works](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-repository-devfile-is-up-to-date-and-works)
- [x] [Sections `What issues does this PR fix or reference` and `How to test this PR` completed](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#sections-what-issues-does-this-pr-fix-or-reference-and-how-to-test-this-pr-completed)
- [x] [Relevant user documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [x] [Relevant contributing documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [x] [CI/CD changes implemented, documented and communicated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#cicd-changes-implemented-documented-and-communicated)

### Reviewers

Reviewers, please comment how you tested the PR when approving it.

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -77,6 +77,11 @@ jobs:
           registry: docker.io
           username: ${{ secrets.DOCKERHUB_USERNAME }}
           password: ${{ secrets.DOCKERHUB_PASSWORD }}
+      -
+        name: Set up yq
+        run: |
+          sudo wget https://github.com/mikefarah/yq/releases/download/v4.7.0/yq_linux_amd64 -O /usr/bin/yq
```
Comment (nickboldt) [inline .github/workflows/release.yml:83]: NOOOO! this is the wrong yq.

**Review by nickboldt — CHANGES_REQUESTED** [review]
mikefarah version has completely different syntax from the pip-installed python wrapper for jq.

File: .github/workflows/release.yml (hunk 1 above)
Comment (nickboldt) [reply .github/workflows/release.yml:83]: I think you want https://github.com/eclipse/che-release/blob/master/.github/workflows/release-orchestrate-overall.yml#L72-L78

**Review by nickboldt — APPROVED** [review]
if this is 100% upstream, then I guess it's OK to use the wrong yq.

**Review by benoitf — APPROVED** [review]

**Pull request merged**